jaxrs-api/pom.xml

```diff
@@ -345,6 +345,11 @@
                 </plugin>
                 <plugin>
                     <!-- This plugin generates the spec.* properties used in maven-bundle-plugin -->
+			
+	            <!--
```

Comment:
@spericas If the plugin is not needed even for an official release, why don't we simply remove it at all?

Reply:
I don't know if it could be of any use to us in the future (don't know much about it to be honest). I'm OK to remove it for now.

Reply:
@spericas the plug-in sets a couple of variables that are used for the OSGi bundle (the `MANIFEST.MF`).

Reply:
I personally do care about OSGi, so -1 for removing it.

(though just the spec plug-in may be worked around, and may not be necessarily needed for OSGI)

Reply:
@arjantijms @mkarg If we can pick up the latest release of the plugin, that would be great.

Reply:
@spericas Indeed, +1 for that. We don't need to file a CQ for that, do we?

Reply:
@spericas Still waiting for Romain.
@arjantijms We do need a CQ for all dependencies not under Eclipse control (which the plugin currently is not AFAIK), but I will take care of that once the solution proofs to work. There is a "fast path" CQ type which allows us to go on without waiting for the outcome.

Reply:
The fixed version of the plugin is the maven.java.net promoted repository.

```xml
            <pluginRepositories>
                <pluginRepository>
                    <id>jvnet-nexus-promoted</id>
                    <url>https://maven.java.net/content/groups/promoted</url>
                    <snapshots>
                        <enabled>false</enabled>
                    </snapshots>
                </pluginRepository>
            </pluginRepositories>
```

Can you test it and let me know ? Once you confirm I'll release it Maven Central and you can drop the use of the repository.

Reply:
@romain-grecourt Currently running a test build on Travis CI...

Reply:
@romain-grecourt Just *using* that plugin does not help. Is there anything I have to explicitly *configure*? It still requests groupId and package names starting with `javax` while it shall request the prefix "jakarta"...

Reply:
I mentioned it in the email thread with Joakim.

See the following diff:

```diff
diff --git a/jaxrs-api/pom.xml b/jaxrs-api/pom.xml
index 884dc52..e311e51 100644
--- a/jaxrs-api/pom.xml
+++ b/jaxrs-api/pom.xml
@@ -352,9 +352,10 @@
                     -->
                     <groupId>org.glassfish.build</groupId>
                     <artifactId>spec-version-maven-plugin</artifactId>
-                    <version>1.2</version>
+                    <version>1.4</version>
                     <configuration>
                         <spec>
+                            <groupIdPrefix>jakarta.</groupIdPrefix>
                             <nonFinal>${non.final}</nonFinal>
                             <jarType>api</jarType>
                             <specVersion>${spec.version}</specVersion>
@@ -646,4 +647,14 @@
         </snapshotRepository>
     </distributionManagement>
 
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>jvnet-nexus-promoted</id>
+                    <url>https://maven.java.net/content/groups/promoted</url>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </pluginRepository>
+            </pluginRepositories>
+
 </project>
```

Reply:
@romain-grecourt Test build on Travis CI was successful! Can you please sync to Maven Central? Thanks a lot for your quick help! :-)

Reply:
Nice, thanks for testing :) It will be in Maven Central sometime in the next 6 hours.

Reply:
@romain-grecourt You're welcome! So finally I found a reason to go to bed now. ;-)

Reply:
Filed [CQ17706](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17706) for `spec-version-maven-plugin:1.4`.